doc/rados/operations/bluestore-migration.rst

```diff
@@ -116,30 +116,52 @@ to evacuate an entire host in order to use it as a spare, then the
 conversion can be done on a host-by-host basis with each stored copy of
 the data migrating only once.
 
-#. Identify an empty host.  Ideally the host should have roughly the
+First, you need an empty host that has no data.  There are two ways to do this:
```

Comment:
How would you feel about making this bold or adding a header like "Identifying an empty host"